frontend/src/libs/components/button/button.tsx

```diff
@@ -3,10 +3,17 @@ import styles from './styles.module.scss';
 type Properties = {
   label: string;
   type?: 'button' | 'submit';
+  style?: 'primary' | 'secondary';
+  isDisabled?: boolean;
 };
 
-const Button: React.FC<Properties> = ({ type = 'button', label }) => (
-  <button type={type} className={styles['submit']}>
+const Button: React.FC<Properties> = ({
+  type = 'button',
+  label,
+  style = 'primary',
+  isDisabled: disabled = false,
```

Comment:
What is the purpose?

Reply:
Missed that when was renaming boolean, already removed